### DISCUSSION TITLE
Missing so-manager_manager.sls

### DISCUSSION THREAD

**Discussion by SanibelJack** [Security-Onion-Solutions/securityonion#1779] [Unsupported Versions] [answered]
DISREGARD THIS POST - MY BAD.  Has anyone experienced this? When I perform a "sudo nano /opt/so/saltstack/local/pillar/minions/so-manager_manager.sls" the file is completely empty. No headers, nothing. Like it's a new file, is this normal? And of course when I try to input some parameters and run so-idstools-restart it errors out.

Thanks

**Comment by dougburks** [comment, marked as answer]
> DISREGARD THIS POST - MY BAD

Marking this as answered.
